### DISCUSSION TITLE
Contar registros en QField

### DISCUSSION THREAD

**Discussion by alexander131290** [opengisch/QField#4100] [Q&A]
¿Es posible contar los registros que cumplen una determinada condición en QField configurando el editor de expresiones en QGIS? He intentado con varias expresiones que funcionan correctamente en QGIS, sin embargo, al cargar el proyecto a QField estas no tienen efecto alguno. Al añadir un nuevo registro a QField se debería detectar si uno de sus campos es igual a un valor específico y asignar una numeración secuencial, añadiendo un prefijo para distinguir del resto, por ejemplo:

Nombre de la capa = Clientes
Campo que debe contenir el valor de compración = Continente (Posibles valores: "América", "Asisa", "Europa")
Campo que debe contener el resultado = Código (Posibles resultados: "AS01", "AS02", "EU01"...)

Si selecciono "Asia" en el campo Continente, el campo código debería mostrar automáticamente "AS01" para el primer registro y "AS02" para el segundo registro. Luego, si selecciono "Europa" el resultado debería ser "EU01", y así sucesivamente.

Gracias anticipadas.

**Comment by suricactus** [comment]
Duplicate of https://github.com/opengisch/QField/discussions/4101
